modules/nw-udn-cr.adoc

```diff
@@ -39,27 +39,68 @@ spec:
 <8> Specifies the IP address management system (IPAM). This field is optional and allowed when `topology` is `layer2`. The
 `subnets` field must be specified when this field is specified. The `Persistent` value specifies that workloads have persistent IP addresses. Assigned by the container network interface (CNI) and used by OVN-Kubernetes to program pod IP addresses. You must not change this for pod annotations.
 
+.. Create a YAML file, such as `my-layer-three-udn.yaml`, to define your request as in the following example:
++
+[source, yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: udn-2-primary # <1>
+  namespace: <some_custom_namespace>
+spec:
+  topology: Layer3 # <2>
+  layer3: # <3>
+    role: Primary # <4>
+    mtu: 1300 # <5>
+    subnets: # <6>
+      - cidr: 10.150.0.0/16
+        hostsubnet: 24
+      - cidr: 2001:db8::/60
+        hostsubnet: 64
+    JoinSubnets: [] # <7>
+----
+<1> Name of your `UserDefinedNetwork` resource. This should not be `default` or duplicate any global namespaces created by the Cluster Network Operator (CNO).
+<2> The `topology` field describes the network configuration, accepted values are `Layer2` and `Layer3`. Specifying a `Layer3` topology type creates a layer 2 segment per node, each with a different subnet. Layer 3 routing is used to interconnect node subnets.
+<3> This field specifies the topology configuration, it can be `layer2` or `layer3`.
+<4> Specifies `Primary` or `Secondary`. `Primary` is the only `role` specification supported in {product-version}.
+<5> The maximum transmission units (MTU) is an optional field. The default value is `1400`. The boundary for IPv4 is 574, and for IPv6 it is 1280.
+<6> For `Layer3` topology types the following specifies config details for the `subnet` field:
++
+* The `subnets` field is mandatory.
+* The type for `subnets` field is `cidr` and `hostsubnet`:
+** `cidr` is the cluster subnet and accepts a string value.
+** `hostSubnet` specifies the nodes subnet prefix that the cluster subnet is split to.
+** For IPv6, only `/64` length is supported for `hostsubnet`.
+<7> It is not recommended to set this field without explicit need and understanding of the OVN network topology. Specifies the routing between different segments within the Layer 3 network. Dual-stack clusters may set 2 subnets, one for each IP family, otherwise only 1 subnet is allowed. This field is only allowed for `Primary` network. When omitted, the platform will choose a reasonable default which is subject to change over time.
+
 .. Apply your request by running the following command:
 +
 [source,terminal]
 ----
-$ oc apply -f my-udn-request.yaml
+$ oc apply -f my-layer-two-udn.yaml
 ----
 
 . Verify that your request is successful by running the following command:
 +
 [source, terminal]
 ----
-$ oc get userdefinednetworks udn-1 -n default -o yaml
+$ oc get userdefinednetworks udn-1 -n some-custom-namespace -o yaml
```

Comment:
I feel like because this is actually a user-replaced value that users could have set to whatever in the yaml files, maybe this should stay in angle brackets:
```suggestion
$ oc get userdefinednetworks udn-1 -n <some_custom_namespace> -o yaml
```

Reply:
Fair enough